modules/protocol-tests/resources/META-INF/smithy/Primitives.smithy

```diff
@@ -0,0 +1,43 @@
+$version: "2"
+
+namespace alloy.test
+
+use alloy#simpleRestJson
+use alloy.test#Primitives
+use smithy.test#httpRequestTests
+use smithy.test#httpResponseTests
+
+apply Primitives @httpRequestTests([
+    {
+        id: "PrimitivesEncodingRequest"
+        protocol: simpleRestJson
+        uri: "/primitive/encoding"
+        method: "POST"
+        body: """
+            {"localTime":"13:26:51.123456789","duration":86400.000000001,"uuid":"51216269-c0c8-454a-871e-329513e54e23","offsetDateTime":"2025-08-15T20:26:51Z","localDate":"2025-08-15"}"""
+        params: {
+            uuid: "51216269-c0c8-454a-871e-329513e54e23"
+            localDate: "2025-08-15"
+            localTime: "13:26:51.123456789"
+            duration: 86400.000000001
+            offsetDateTime: 1755289611
```

Comment:
The smithy validator only allows to specify a timestamp in epoch seconds, which is what `OffsetDateTime` is under the hood. With this restriction we can't specify an offset for the `OffsetDateTime` for the expected body. I could create a trait to specify a default offset to use, but would like to get others opinions first.

Reply:
Ah, that's a tricky one. Where does this cause a failure on the smithy4s side of things? Is it when we try to take `params` to decode them? Or is it later on?

Reply:
The error is present in here so it's part of the `httpRequestTests` validator when the model is loaded. Here's a link to a failed build with the error for commit `fd4d48b` https://github.com/disneystreaming/alloy/actions/runs/17082654806/job/48440082555#step:7:86